examples/notebooks/tour-of-beam/getting-started.ipynb

```diff
@@ -0,0 +1,1021 @@
+        "> the `PCollection` elements might live in multiple worker machines.\n",
+        "\n",
+        "We'll do a little trick to print the elements, but we'll explain it shortly."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "XAke4d5lV5f5",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "8e032211-c70a-4a89-9904-58d6fa2c8cb7"
+      },
+      "source": [
+        "import apache_beam as beam\n",
+        "\n",
+        "inputs = [0, 1, 2, 3]\n",
+        "\n",
+        "# Create a pipeline.\n",
+        "with beam.Pipeline() as pipeline:\n",
+        "  # Feed it some input elements with `Create`.\n",
+        "  outputs = (\n",
+        "      pipeline\n",
+        "      | 'Create initial values' >> beam.Create(inputs)\n",
+        "  )\n",
+        "\n",
+        "  # `outputs` is a PCollection with our input elements.\n",
+        "  # But printing it directly won't show us its contents :(\n",
+        "  print(f\"printing won't show us the elements: {outputs}\\n-----\\n\")\n",
+        "\n",
+        "  # We can only access the elements through another transform.\n",
+        "  # Don't worry if you don't know what's happening here,\n",
+        "  # we'll get to it just next :)\n",
+        "  outputs | beam.Map(print)"
+      ],
+      "execution_count": 26,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "printing won't show us the elements: PCollection[[26]: Create initial values/Map(decode).None]\n",
+            "-----\n",
+            "\n",
+            "0\n",
+            "1\n",
+            "2\n",
+            "3\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "FY9TP3Tw5eZt",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Transforming data\n",
+        "\n",
+        "Apache Beam is designed with a [functional paradigm](https://en.wikipedia.org/wiki/Functional_programming).\n",
+        "This means that, instead of _loops_, it uses `PTransform`s alongside with _functions_ to process each element in a `PCollection`.\n",
+        "\n",
+        "Let's go through some of the most common and basic data transforms in Beam."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "UMKgi9LD6-xb",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Map -- *one to one*\n",
+        "\n",
+        "Let's say we have some elements and we want to do something with each element.\n",
+        "\n",
+        "We want to `map` a function to each element of the collection.\n",
+        "\n",
+        "`map` takes a _function_ that transforms a single input `a` into a single output `b`.\n",
+        "\n",
+        "> 🐌 -- For example, we want to multiply each element by 2."
```

Comment:
What does the 🐌 indicate? I got the sense that "this might be a slow step" from seeing that.

Reply:
It was just a cute emoji, but that makes sense. I'm replacing them with ▶️